Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetVerticalUITest.swift

```diff
@@ -30,6 +30,7 @@ class PaymentSheetVerticalUITests: PaymentSheetUITestCase {
         settings.mode = .setup
         settings.customerMode = .new
         settings.currency = .eur
+        settings.customerKeyType = .legacy // TODO: Change to customerSessions when adding save checkbox for SEPA
```

Comment:
Do we have a JIRA for this? Seems likely we'll forget to do this, same with the other TODOs.

Reply:
Added 3 new JIRAS.